modules/API/API.js

```
@@ -302,8 +304,10 @@ function initCommands() {
 
             const activeSession = getActiveSession(state, mode);
 
-            if (activeSession && activeSession.id) {
-                conference.stopRecording(activeSession.id);
+            if (activeSession && (activeSession.id || activeSession.queueID)) {
+                conference.stopRecording(activeSession.id, activeSession.queueID).catch(() => {
```

same here, why swallowing unhandled rejections?